tests/migration/openldap_to_389ds.pm

```diff
@@ -23,58 +21,12 @@ sub run {
     my $password = $testapi::password;
     select_serial_terminal;
 
-    # Install 389-ds and sssd on client
-    zypper_call("in 389-ds sssd sssd-tools");
+    # Install 389-ds
+    zypper_call("in 389-ds");
     zypper_call('info 389-ds');
-    # Install openldap since we need use slaptest tools
-    if (is_sle) {
-        my $openldap = get_required_var("OPENLDAP_PKG");
-        assert_script_run('wget ' . $openldap, 200);
-        my @ldap = split('/', $openldap);
-        assert_script_run 'rpm -i ' . $ldap[-1];
-    } else {
-        zypper_call("in openldap2");
-    }
-    zypper_call("in sssd-ldap openldap2-client");
-
-    # Disable and stop the nscd daemon because it conflicts with sssd
-    disable_and_stop_service("nscd");
-
-    # On newer environments, nsswitch.conf is located in /usr/etc
-    # Copy it to /etc directory
-    script_run 'f=/etc/nsswitch.conf; [ ! -f $f ] && cp /usr$f $f';
-    # Configure nsswitch with sssd
-    assert_script_run "sed -i 's/^passwd:.*/passwd: compat sss/' /etc/nsswitch.conf";
-    assert_script_run "sed -i 's/^group:.*/group: compat sss/' /etc/nsswitch.conf";
-    assert_script_run "cat /etc/nsswitch.conf";
-
-    # Prepare test env
-    assert_script_run "cd; curl -L -v " . autoinst_url . "/data/openldap_to_389ds > openldap_to_389ds.data && cpio -id < openldap_to_389ds.data && mv data test && ls test";
-    assert_script_run("cd test");
-
-    # We need start openldap to kick out date base file which stored in directory
-    assert_script_run "mkdir /tmp/ldap-sssdtest";
-    if (is_tumbleweed) {
-        assert_script_run "sed -i -e '/cachesize/d' ./slapd.conf";
-        assert_script_run "sed -i -e 's/hdb/mdb/g' ./slapd.conf";
-        # for openqa debug
-        permit_root_ssh;
-    }
-    assert_script_run "cat ./slapd.conf";
-    assert_script_run "slapd -h 'ldap:///' -f slapd.conf";
-
-    assert_script_run "ldapadd -x -D 'cn=root,dc=ldapdom,dc=net' -wpass -f db.ldif";
-    assert_script_run "killall slapd";
-    assert_script_run "ps -aux | grep slapd";
-
-    # setup sssd
-    assert_script_run "cp ./sssd.conf /etc/sssd/sssd.conf";
-    systemctl("stop sssd");
-    assert_script_run "rm -rf /var/lib/sss/db/*";
-    systemctl("restart sssd");
-    systemctl("status sssd");
 
     # Prepare data file for migration
```

Comment:
prepare data section should move to openldap_configuration

Reply:
thanks for taking a look. Are you referring to the password replacement bellow? That would make sense. I moved this to the part after migration because dscreate is included in 389-ds and not in openldap2